Makefile

```diff
@@ -23,6 +23,8 @@
 
 SRCS := $(wildcard src/*.cpp)
 TESTS := $(wildcard test/*.cpp)
+MANS := $(wildcard documentation/*.man)
+MAN_HS := $(patsubst documentation/%.man, documentation/man-%.h, $(MANS))
```

Comment:
aren't the arguments to patsubst still in the wrong order?

Reply:
It was giving me this error when I did it the way you initially commented:
make: *** No rule to make target 'documentation/%.man', needed by 'src/main.o'.  Stop.

I found this thread that made me think to switch the order
https://stackoverflow.com/questions/14028268/proper-method-for-wildcard-targets-in-gnu-make

I'm pretty sure it works as expected now

Reply:
Seems you're right, I must have been confused